source/2019-12-13-the-ember-times-issue-128.md

```diff
@@ -46,13 +49,35 @@ Want to contribute to the cheat sheet yourself? Check out [the project on Github
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Introduce Adaptive Loading to Your App with Ember Device 📶](https://twitter.com/_gokatz/status/1201534724696494081)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+With [Ember Device](https://github.com/gokatz/ember-device), you can **progressively deliver rich content and powerful web experiences**. Thanks to [Gokul Kathirvel (@gokatz)](https://github.com/gokatz), you can **provide as much value to users with low-end devices** as users with high-end devices. 💯
```

Comment:
I'm not entirely sure how this works as a non-native speaker, but I'm wondering if it'd be

```suggestion
With [Ember Device](https://github.com/gokatz/ember-device), you can **progressively deliver rich content and powerful web experiences**. Thanks to [Gokul Kathirvel (@gokatz)](https://github.com/gokatz), you can **provide as much value to users with low-end devices** as to users with high-end devices. 💯
```

What do you think?

Reply:
No worries. I wasn't quite sure myself when I wrote that. I think your sentence sounds better, thanks!